src/js/base/config.js

```diff
@@ -46,3 +46,60 @@ ripe.Ripe.prototype.hasStrategy = function(strategy) {
     const strategies = (this.loadedConfig && this.loadedConfig.strategies) || ["prc"];
     return strategies.includes(strategy);
 };
+
+/**
+ * Retrieves the initials config with the specified profiles in the config. If the
+ * `profiles` argument is provided those profiles will be also used to compute the
+ * initials config.
+ *
+ * @param {Object} config The model's config.
+ * @param {Array} profiles The list of profiles to use.
+ * @returns {Object} The computed initials config based on the config profiles and
+ * specified profiles.
+ */
+ripe.Ripe.prototype.initialsConfig = function(config, profiles = []) {

```

Comment:
Since we are not removing the remote method I'm assuming we maintaining the `getInitialsConfigP` method to be used as an alternative remote. The so called `remote = true` approach we discussed.

Reply:
Yes, this is just an alternative that runs locally instead of doing the remote call